### ISSUE TITLE
Oregon historic commercial catch

### ISSUE THREAD

**Issue by brianlangseth-NOAA** [pfmc-assessments/canary_2023#33]
With regards to the Oregon historic commercial catch time series, we have discovered that during the last assessment for black rockfish, trawl species comps were determined to be too variable from year to year, and so they were aggregated to stabilize the historic trawl landings. We are likely going to be using the same methodology for this assessment. For the last Canary assessment, no changes were made to the historic time series, despite being based on the same data. 

Here's a snapshot of our species comps for Canary during the period in question. These are annually aggregated comps (which is all we have, unfortunately, the raw comps are no longer available). Canary was one of the primary trawl species at the time, which gives me more confidence in the species comps than it does for Black.  (Ignore 1C and 3B below, they're outside of OR).
![image](https://user-images.githubusercontent.com/80843508/213246497-678f9829-7d16-417d-96b4-6d74f880e3de.png)

I wanted to make you guys aware of this, but ultimately, I would still recommend that we stick with the historic time series (identical to the last Canary assessment) but include a recommendation to revisit the historic trawl landings in Oregon (we are planning to include a similar recommendation in the Black rockfish assessment) so that this could get sorted out in a comprehensive fashion.  What are your thoughts on this?

_Originally posted by @aliwhitman in https://github.com/pfmc-assessments/canary_2023/issues/9#issuecomment-1387430771_

**Comment by brianlangseth-NOAA** [comment]
Trying something new to minimize long individual issues so pulling out this topic as a separate issue.

@aliwhitman I have no issue with using the same historical time series as the last assessment. I dont fully understand the problem though, so it is hard for me to comment on it. I imagine it would be easier to discuss via phone instead of text. 

@tsoutt mentioned an issue with trawl catch out of astoria that is from Washington waters but is landed in Oregon. Is this related to that?

**Comment by aliwhitman** [comment]
So both the variable trawl species comp issue and the location of the trawl catches are related in that they were both issues that had to be addressed for the last black rockfish assessment. So sort of.  Sorry if that's unclear. 

With the last black rockfish assessment, logbook data was used to separate Astoria (3A) landings between OR and WA, as it had separate models for each state.  It is not an issue (unless I am missing something) for this assessment as a coastwide assessment, unless I am missing something! If we do need to separate Canary trawl landings from 3A, we could possibly use the same data source, but I'm still trying to track down the original files used. 

Happy to discuss offline too!

**Comment by tsoutt** [comment]
I am wondering how it will affect the assessment results if there are spatial structures implemented in the coastwide assessment. Also, how will GMT use the results for sector/state catch allocation?

**Comment by aliwhitman** [comment]
In my attempt to understand more of the issue here, I looked at trawl logbook data for Area 3A from 1995 - 2022 and found that in years without the overfished designation, roughly 2/3 of the canary (in terms of lbs hailed) landed into Astoria was from WA waters (areas 29/30).  This 2/3 is on the order of 75k pounds annually on average, so a significant source of mortality. I'm happy to provide summaries of the logbook data if folks would like to look at them in more detail (we'd have to amend the NDA if you'd like the raw data but happy to do that too of course).  

I know that we're still sorting out whether moving fish across state lines would matter for this assessment but wanted to have an idea of the scale of the issue.  Please let me know your thoughts on this. 

Also, I could provide an unmodified version of the Oregon's historic commercial catches as a preliminary catch history to have for the pre-assessment workshop (that's all ready to go!).

**Comment by brianlangseth-NOAA** [comment]
Having an unmodified version of the historical commercial catch is a good first step. Gives us a chance to continue with modeling. So yes, having that would be helpful. Does the change in shrimp trawl from trawl gears to non-trawl gears affect this? 

As for effect on our model, the effect will be less than if we had three separate models, and less (and possibly not an issue at all) if we exclude partitioning recruitment. IF we go with a spatially structured population (include recruitment apportionment) I think that moving catches (and comps) between them will affect recruitment deviation estimates. I dont know how much it will affect it, but there should be a modeling effect. The effect on relevant coastwide management measures should be less though since these are provided coastwide. All this to say is that we would need to test the effect. 

As for the effect on GMT state allocations, that would be something to raise with them. That seems like a question outside the STAT. 

To test the effect on the model, we would need a modified version. To be complete we would want both catches and bio data, if that data exists, identified. Once that is in place, wroking it into the model should be straightforward.

**Comment by tsoutt** [comment]
Will any of the ODFW catch reconstruction of URCK and UPOP be used in this canary assessment? I was able to locate our catch reconstruction work conducted to support 2017 assessment cycle. If ODFW catch reconstruction for URCK and UPOP will be used, maybe we do the same for WA landings. 

Brian & Kiva - these are based on the spp comps not available in PacFIN.

**Comment by brianlangseth-NOAA** [comment]
@tsoutt Im not sure what you mean by reconstructions. Based on issue #9 Oregon will use their own expansions. Presumably they will apply those to their historical reconstructions but @aliwhitman is the best person to confirm. 

I do not have an issue with WA developing their own URCK and UPOP catch expansions, as catch streams are finalized by the states, provided those are given to us by the data deadline (though hopefully earlier). If that is done, I will expect WA to defend that choice to the council/STAR panel should questions arise. Would this be a canary only thing, or are you thinking of applying that approach to other species this cycle?

**Comment by aliwhitman** [comment]
Yes, we are recommending using our Oregon URCK/POP reconstruction, which Theresa is correct - It's not on PacFIN yet. I am providing directly to the STAT.  The unmodified preliminary commercial catch time series that I provided on 2/6 includes this reconstruction (years 1987 - 1999) already. Happy to answer questions on that.

**Comment by tsoutt** [comment]
Thank you, Ali, for confirming it. 
Brian and Kiva - I will email you the reconstructed catch history this week.
